guides/common/modules/proc_applying-errata-to-a-host-sles15.adoc

```diff
@@ -0,0 +1,30 @@
+[id="Applying_Errata_to_a_Host_Running_SLES_15_{context}"]
+= Applying Errata to a Host Running SLES 15
+
+Use these procedures to review and apply errata to a host running SUSE Linux Enterprise Server 15.
```

Comment:
This implies that they'd need to do two, while there are just alternative options for web ui and cli. I'd stick with the singular. If you feel strongly about this, perhaps add a second sentence saying, you can find steps for the corresponding CLI procedure after the web UI-based procedure.... 
```suggestion
Use this procedure to review and apply errata to a host running SUSE Linux Enterprise Server 15.
```